vision/manipulation/super_resolution/README.md

```diff
@@ -0,0 +1,58 @@
+# SuperResolution
+
+## Use cases
+The Super Resolution machine learning model sharpens and upscales the input image to refine the details and improve quality.
+
+## Description
+Super Resolution uses efficient  [Sub-pixel convolutional layer](https://arxiv.org/abs/1609.05158) described for increasing spatial resolution within network tasks. By increasing pixel count, images are then clarified, sharpened, and upscaled without losing the input image’s content and characteristics. 
+
+## Model
+
+ |Model      |Download  |Checksum|Download (with sample test data)| ONNX version |
+|-------------|:--------------|:--------------|:--------------|:--------------|
+|Super_Resolution v1|    [240 KB](super_resolution_tutorial.onnx)  |[MD5](super_resolution-md5.txt)  |  [7.6 MB](super_resolution_test_image.gz) |  1.5.0  |
+
+## Inference
+
+
+### Input 
+Image file can be jpg, png, and jpeg and its input sizes are dynamic. The inference was done using jpg image.
+
```

Comment:
It maybe useful to add a paragraph up here to explain at a high-level how super resolution output is obtained. We should clearly mention that the image format is changed to YCbCr and then the intensity channel (Y) is the only one that is passed through the superresolution model to get upscaled intensity image. The color channels undergo a simple bicubic interpolation based upscaling and then as the last steps they are all combined and converted back to RGB format.

Reply:
Is it better to add this piece of information to the preprocessing stage rather than the input stage?